<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotpointPanel</class>
 <widget class="QFrame" name="HotpointPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>675</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>675</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <property name="horizontalSpacing">
    <number>12</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="HotPointBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>270</width>
       <height>629</height>
      </size>
     </property>
     <property name="title">
      <string>Hotpoint/POI Setup</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_59">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <widget class="QGroupBox" name="hpPositionInputBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>160</height>
         </size>
        </property>
        <property name="title">
         <string>Position</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_39">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_hotPoint_current">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Set Current Position As Hotpoint</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_hp_data">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Reset and Input New Hoptpoint Position</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_34">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_41">
             <item>
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>Latitude (Deg)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_hp_la">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_42">
             <item>
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>Longitude (Deg)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_hp_lo">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_43">
             <item>
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>Altitude (Meters)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_hp_al">
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="hpParametersInputBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Mission Parameters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_51">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_39" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Radius</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_hp_ra">
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_hp_setRadius">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>set</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_48">
           <item>
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>Yaw Rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_hp_pa">
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="text">
              <string>15</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_hp_setPal">
             <property name="text">
              <string>set</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_53">
           <item>
            <widget class="QPushButton" name="btn_hp_setYaw">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>reset</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_hp_yaw">
             <item>
              <property name="text">
               <string>Auto Yaw</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Align Inside</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Align Outside</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>User define</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Static</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cb_hp_cl">
             <property name="text">
              <string>clockwise</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_37">
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_hotPoint_start">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_hotPoint_stop">
          <property name="text">
           <string>stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_hp_pause">
          <property name="text">
           <string>pause</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_40">
        <item>
         <widget class="QPushButton" name="btn_camera_send_8">
          <property name="text">
           <string>check</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_camera_send_7">
          <property name="text">
           <string>reset yaw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
